5. Administrator Guides/Integrations/Google-Calendar.md

```diff
@@ -0,0 +1,12 @@
+# Google Calendar
+
+[Google Calendar](https://calendar.google.com/) is calendar from Google company. You can easily organize your daily tasks or meetings and share with your co-workers, etc..
+This integration uses Rocket.chat incoming webhook.
```

Comment:
To keep our official brand/CI, I would capitalize the "C" of `chat` in this case: Rocket.Chat